src/diem/offchain/client.py

```diff
@@ -144,15 +144,14 @@ async def send_request(
             http_header.X_REQUEST_SENDER_ADDRESS: request_sender_address,
         }
         url = f"{base_url.rstrip('/')}/v2/command"
-        async with aiohttp.ClientSession() as session:
-            async with session.post(url, data=request_bytes, headers=headers) as response:
-                if response.status not in [200, 400]:
-                    response.raise_for_status()
-
-                cmd_resp = _deserialize_jws(await response.read(), CommandResponseObject, public_key)
-                if cmd_resp.status == CommandResponseStatus.failure:
-                    raise CommandResponseError(cmd_resp)
-                return cmd_resp
+        async with self.jsonrpc_client._session.post(url, data=request_bytes, headers=headers) as response:
+            if response.status not in [200, 400]:
```

Comment:
nit: can just be `response.status != 400:`, the `200` won't raise in `raise_for_status`

Reply:
I think `response.status not in [200, 400]` is easier for reader to understand.